posthog/api/feature_flag.py

```diff
@@ -283,6 +283,15 @@ def update(self, instance: FeatureFlag, validated_data: Dict, *args: Any, **kwar
             raise exceptions.ValidationError(
                 "Cannot delete a feature flag that is in use with early access features. Please delete the early access feature before deleting the flag."
             )
+
+        if "get_filters" in validated_data:
```

Comment:
can you move this check to `validate_filters` instead? All similar checks exist there